src/src/modules/auth/auth.service.ts

```diff
@@ -209,21 +219,7 @@ export class AuthService {
 		const username = req.username;
 		const reason = req.reason;
 
-		//find mobile by username
-		let query = {
-			query: `query MyQuery2 {
-                users(where: {username: {_eq: ${username} }}) {
-                  keycloak_id
-                  last_name
-                  id
-                  first_name
-                  mobile
-                }
-              }`,
-		};
-		const userRes: any = await this.hasuraService.postData(query);
-		console.log('userRes', userRes);
-
+		const userRes = await this.getUserByusername(req);
```

Comment:
not using renamed name?

Reply:
we have created new function getUserByusername and we are calling that so i had change there